tests/utils/object_store/test_uc_object_store.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+from pathlib import Path
+from unittest import mock
+from unittest.mock import ANY, MagicMock
+
+import pytest
+from torch.utils.data import DataLoader
+
+from composer.loggers import RemoteUploaderDownloader
+from composer.trainer import Trainer
+from composer.utils import UCObjectStore
+from composer.utils.object_store.object_store import ObjectStoreTransientError
+from tests.common import RandomClassificationDataset, SimpleModel
+
+
+@pytest.fixture
+def ws_client(monkeypatch):
+    mock_files = MagicMock()
+    mock_ws_client = MagicMock()
+    monkeypatch.setattr(mock_ws_client, 'files', mock_files)
+    return mock_ws_client
+
+
+@pytest.fixture
+def uc_object_store(ws_client, monkeypatch):
+    db = pytest.importorskip('databricks.sdk', reason='requires databricks')
+
+    monkeypatch.setenv('DATABRICKS_HOST', 'test-host')
+    monkeypatch.setenv('DATABRICKS_TOKEN', 'test-token')
+    with mock.patch.object(db, 'WorkspaceClient', lambda: ws_client):
+        yield UCObjectStore(path='Volumes/catalog/schema/volume/path/')
+
+
+@pytest.mark.remote  # databricks auth is hard to set up on github actions
```

Comment:
this won't pass. "remote" means "run only on github actions, not on local machine". Could you make a jira for getting databricks auth into github actions? you can just skip this test for now, and we will rely on the manual test of it that you did

Reply:
Ah I see. Added a ticket here: https://databricks.atlassian.net/browse/MCLOUD-3169 and skipped the test for now.